### DISCUSSION TITLE
Facing Errors while installing

### DISCUSSION THREAD

**Discussion by PYIArun** [kom-senapati/bot-verse#1] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/905c8084-3241-4280-8318-4d10f0140a40)

I was progressing with the installation, and correctly done till this point. But with the command "flask db init", it causing some errors. Any solution for that.?

**Comment by PYIArun** [comment]
[Fixed] Removing the "from crypt import methods" from the routes.py file, works for me.

**Reply by k0msenapati** [reply]
It works well in linux envs like codespace. Can you try that after removing it in codespace and check if it runs

**Reply by PYIArun** [reply]
Yeah, i just removed that line, i mentioned in my above reply and it runs.

**Reply by k0msenapati** [reply, marked as answer]
Ok raise one issue for this then you can do PR corresponding to that issue.
